layouts/_default/single.html

```diff
@@ -6,9 +6,9 @@ <h1 class="post-title">
   <div class="post-meta">
     {{- if .Date -}}
       <time class="post-date">
-        {{- .Date.Format "2006-01-02" -}}
-        {{- if $.Site.Params.showLastUpdated -}}
-          [{{- or $.Site.Params.updatedDatePrefix "Updated" -}} :: {{- .Lastmod.Format "2006-01-02" -}}]
+        {{- partial "post-date" . -}}
+        {{- if .Lastmod -}}
+            &nbsp;[{{- partial "post-date" . (dict "Date" .Lastmod) -}}]
```

Comment:
This won't work... You are passing the original context and `(dict ... )` part is skipped by Hugo (AFAIR).
Also I made a mistake thinking that passing only one value to this partial solves the issue, but unfortunately I missed that also need other props from the site context...

So the new idea is to replicate this partial as `post-lastmod` and just use it separately with `.Lastmod` key inside instead of `.Date`...

Reply:
For example:

<img width="2560" alt="image" src="https://github.com/user-attachments/assets/6b9ed9f4-b31d-4e97-b672-660abf04fe5a">

Reply:
Funky, guess you're right. For some reason it seemed to be working when I tested it on my computer but I guess for some reason it doesn't work at the end of my CICD pipeline. I will split it and get back to you today.